llvm/docs/CommandGuide/llvm-profdata.rst

```diff
@@ -475,3 +478,40 @@ EXIT STATUS
 
 :program:`llvm-profdata` returns 1 if the command is omitted or is invalid,
 if it cannot read input files, or if there is a mismatch between their data.
+
+.. _profdata-canonical:
+
+SAMPLE PROFILE CANONICAL FUNCTION NAME
+--------------------------------------
+
+The canonical name of a (possibly inlined) function in sample profile is
+defined as the following:
```

Comment:
A few examples would be more clear and user friendly than the definitions below.  

(The definitions of text profile format is in SampleProfReader.h, even there we didn't use strict definitions.)